src/ndi-source.cpp

```diff
@@ -547,6 +554,7 @@ void *ndi_source_thread(void *data)
 			}
 
 			// Deactivate the source output video texture when using Audio only
+			// ISSUE: Do we need to do both this and in `ndi_source_update`?
 			if (recv_desc.bandwidth ==
```

Comment:
This is needed to "clean" the Source Content when switching to Audio Only i we want to keep the original "user experience" behavior as in :

- Open NDI Source settings
- Change bandwidth to audio only
> as soon as this is selected > it will "clear" the source content/frame
- On "save" > this should be calling the ndi_source_update ()

Ths goes along side with the UX we want for users : any changes made in the source settings shoud apply right away or only on save?
Default today is expected as : "apply as soon as changed". Commit to config on save. Revert on Cancel.

If we want to keep this > This should stay here.